Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -695,9 +695,29 @@ def test_log
         system "git", "commit", "--allow-empty", "-m", "This is a test commit"
       end
     end
+
+    core_tap = CoreTap.new
```

Comment:
I think the `test_$CMDNAME_$VARIANT` format in addition to `test_$CMDNAME` for the integration tests gives us a bit more flexibility, particularly in cases like these where the two tests in the method are completely orthogonal and combining them in a single method just makes them harder to read due to the interleaving of setup, test, and teardown code. (This also aids with isolating tests from each other, so they don't accidentally influence each other.)

That said, I agree that with some refactoring the code in this particular test method could become simple enough to make a split unnecessary.